src/components/Footer/Footer.module.scss

```diff
@@ -18,7 +18,7 @@
     list-style-type: none;
   }
 
-  h4 {
+  .headline {
     margin-block-end: ds.$spacing-16;
```

Comment:
Why not use the heading mixin here on a paragraph?
http://localhost:6006/?path=/docs/components-typography-heading--documentation